client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -0,0 +1,200 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormToggle from 'components/forms/form-toggle';
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackModuleActive } from 'state/selectors';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+
+class CustomContentTypes extends Component {
+	isFormPending() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+		} = this.props;
+
+		return isRequestingSettings || isSavingSettings;
+	}
+
+	renderToggle( name, isDisabled, label ) {
+		const { fields, handleToggle } = this.props;
+		return (
+			<FormToggle
+				className="custom-content-types__module-settings-toggle is-compact"
+				checked={ !! fields[ name ] }
+				disabled={ this.isFormPending() || isDisabled }
+				onChange={ handleToggle( name ) }>
+				<span className="site-settings__toggle-label">
+					{ label }
+				</span>
+			</FormToggle>
+		);
+	}
+
+	renderHeader() {
+		const {
+			onSubmitForm,
+			isSavingSettings,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<SectionHeader label={ translate( 'Custom Content Types' ) }>
+				<Button
+					compact
+					primary
+					onClick={ onSubmitForm }
+					disabled={ formPending }
+				>
+					{ isSavingSettings
+						? translate( 'Saving…' )
+						: translate( 'Save Settings' )
+					}
+				</Button>
+			</SectionHeader>
+		);
+	}
+
+	renderModuleToggle() {
+		const {
+			siteId,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<div>
+				<div className="custom-content-types__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/custom-content-types' } target="_blank">
+							{ translate( 'Learn more about Custom Content Types' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				<JetpackModuleToggle
+					siteId={ siteId }
+					moduleSlug="custom-content-types"
+					label={ translate( 'Display different types of content on your site with custom content types.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Use these settings to display different types of content on your site.' )` *ES Score: 5.13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).